server/game/cards/06-WoE/GreyAbess.js

```diff
@@ -0,0 +1,18 @@
+const Card = require('../../Card.js');
+
+class GreyAbess extends Card {
```

Comment:
Either the comment or the implementation is incorrect

Reply:
Yeah, comment is a copy/paste relic. Comments are the worst 😅.

If it’s alright I’m happy to put a pic of the card in each PR instead.

Reply:
Updated 👍